src/nimble.nim

```diff
@@ -639,22 +646,24 @@ proc processLockedDependencies(pkgInfo: PackageInfo, options: Options, onlyNim =
 
   let developModeDeps = getDevelopDependencies(pkgInfo, options)
 
+  var res: seq[PackageInfo]
+
   for name, dep in pkgInfo.lockedDeps.lockedDepsFor(options):
-    if onlyNim and not name.isNim:
-      continue
     if developModeDeps.hasKey(name):
-      result.incl developModeDeps[name][]
+      res.add developModeDeps[name][]
     elif isInstalled(name, dep, options):
-      result.incl getDependency(name, dep, options)
+      res.add getDependency(name, dep, options)
     elif not options.offline:
       let
         downloadResult = downloadDependency(name, dep, options)
-        dependencies = result.toSeq.filterIt(dep.dependencies.contains(it.name))
-      result.incl installDependency(pkgInfo.lockedDeps.lockedDepsFor(options).toSeq.toTable,
-                                    downloadResult, options, dependencies)
+        dependencies = res.toSeq.filterIt(dep.dependencies.contains(it.name))
```

Comment:
Since `res` is already a sequence here, the `toSeq` call is redundant.

Reply:
Fixed.